tests/Behat/DateFieldContext.php

```diff
@@ -119,13 +119,74 @@ public function fillDateRangeSelectListField(string $field_item, string $field_g
   }
 
   /**
-   * Set the date and time value of a date list widget.
+   * Check values for list date range fields.
    *
-   * When I set "Field" to the date "22-02-2019"
-   * When I set "Field" to the date "22-02-2019 14:30" using format "d-m-Y H:i"
+   * Then datetime "15 1 2020 12 30" is selected for "Start date" of "Datetime"
```

Comment:
Keep the previous format. Also avoid passive forms as much as possible:

```suggestion
   * When I set "22-02-2019 14:30" as "start date" for "Field name"
```